Gemfile.lock

```diff
@@ -13,13 +13,13 @@ GEM
     docile (1.1.5)
     json (1.8.6)
     method_source (0.8.2)
-    mini_portile2 (2.4.0)
-    nokogiri (1.10.4)
-      mini_portile2 (~> 2.4.0)
-    openxml-package (0.2.3)
+    mini_portile2 (2.1.0)
+    nokogiri (1.6.8.1)
```

Comment:
Interesting that the version of `nokogiri` went _backwards_ here. What's up with that??